pkg/rancher-desktop/utils/_demo_metadata.js

```diff
@@ -398,7 +398,7 @@ export default {
+          'com.docker.extension.detailed-description': '\u003Cp\u003EThis Docker Desktop extension allows you to browse and filter logs from running containers.\u003C/p\u003E \u003Ch2 id=-features\u003E✨ Features\u003C/h2\u003E \u003Cul\u003E \u003Cli\u003EBrowse logs for running containers\u003C/li\u003E \u003Cli\u003ESelect individual containers for which logs are displayed\u003C/li\u003E \u003Cli\u003EInfinite scrolling\u003C/li\u003E \u003Cli\u003EScrolling locked to bottom by default\u003C/li\u003E \u003Cli\u003ESearch logs with regular expressions or exact match\u003C/li\u003E \u003Cli\u003E\u0026quot;Sticky\u0026quot; search filters\u003C/li\u003E \u003Cli\u003EContextual keyword filters\u003C/li\u003E \u003Cli\u003EAutomatic light/dark mode\u003C/li\u003E \u003C/ul\u003E \u003Ch2 id=-tips\u003E🎩 Tips\u003C/h2\u003E \u003Cul\u003E \u003Cli\u003EYou can select multiple containers by pressing \u003Ccode\u003EShift\u003C/code\u003E while clicking on a container name in the sidebar\u003C/li\u003E \u003Cli\u003EYou can select a container by clicking a container name in the logs output pane\u003C/li\u003E \u003Cli\u003EYou can pause logs following by clicking the Play/Pause button at the top left of the logs pane\u003C/li\u003E \u003Cli\u003EYou can make filter terms \u0026quot;sticky\u0026quot; by pressing \u003Ccode\u003EEnter\u003C/code\u003E in the search box\u003C/li\u003E \u003Cli\u003EYou can use keyword filters (similar to Slack search) in the search box:\u003Cul\u003E \u003Cli\u003E\u003Ccode\u003Esince:\u0026lt;duration\u0026gt;\u003C/code\u003E: restrict the search to the log lines outputted in the past \u003Ccode\u003E\u0026lt;duration\u0026gt;\u003C/code\u003E time window (e.g.: \u003Ccode\u003Esince:2m30s\u003C/code\u003E)\u003C/li\u003E \u003Cli\u003E\u003Ccode\u003Efrom:\u0026lt;time\u0026gt;\u003C/code\u003E, \u003Ccode\u003Eto:\u0026lt;time\u0026gt;\u003C/code\u003E: restrict the search to the log lines outputted in the time window with the given boundaries\u003C/li\u003E \u003C/ul\u003E \u003C/li\u003E \u003Cli\u003EBy default search uses case-insensitive regular expressions; if you want you can perform an exact match by wrapping your search query with double quotes (e.g. \u003Ccode\u003E\u0026quot;foo\u0026quot;\u003C/code\u003E)\u003C/li\u003E \u003C/ul\u003E',
```

Comment:
```diff
-outputed
+outputted
```

Reply:
I personally would prefer `output`

From https://www.thefreedictionary.com/output :
```
Past participle: output
```

Reply:
Yeah, I do too 😊, but I was being lazy. Fixed.

Reply:
FWIW, this came from the labels on `docker/logs-explorer-extension:0.1.4` (the image). What we should do here instead is to update the metadata (it's been changed since, and no longer contains the problematic word).